pandas/tests/groupby/aggregate/test_other.py

```diff
@@ -328,7 +328,7 @@ def test_series_agg_multi_pure_python():
          'F': np.random.randn(11)})
 
     def bad(x):
-        assert (len(x.base) > 0)
+        assert (len(x.values.base) > 0)
```

Comment:
Not really sure what the purpose of this assert actually is (was introduced in https://github.com/pandas-dev/pandas/commit/71e9046c52246535d4db1f350e82c3a84d748f88)